users/views.py

```diff
@@ -14,3 +21,18 @@ def post(self, request):
             content = {'message': 'wrong google token / this google token is already expired.'}
             return Response(content)
         return Response(r)
+
+        try:
+            user = User.objects.get(email=data['email'])
```

Comment:
```suggestion
  user = User.objects.get(email=request.data.get('email'))
```
Also remove this line.

Reply:
Still this change won't work, because `User.objects` is invalid.